ui-components/src/helpers/tableSummaries.tsx

```diff
@@ -5,7 +5,6 @@ import { GroupedTableGroup } from "../tables/GroupedTable"
 
 export const getSummaryRow = (
   totalAvailable: number,
-  totalCount: number,
```

Comment:
Is this helper ours (and not yet contributed upstream)?

Reply:
The helper file (`ui-components/src/helpers/tableSummaries.tsx`) is originally Exygy's. But we added `totalCount` a while ago (https://github.com/CityOfDetroit/bloom/pull/189) and never committed that upstream. So this current PR brings our `tableSummaries.tsx` closer in line with Exygy's (since it partially reverts that #189 ).

Having written that, I'm wondering if it's worthwhile doing a full revision of #189. I don't have strong feelings. I'll submit this current PR now, but let me know if you feel that we should fully revert #189.